### DISCUSSION TITLE
How to backup native app saves?

### DISCUSSION THREAD

**Discussion by Softhook** [eduardofilo/RG350_adam_image#146] [Ports]
I notice that PyBackup doesn't backup my native Mario N64 save. 
Any advice on how could I do that in the future?

**Comment by eduardofilo** [comment]
Yes. Add this to the end of file `/media/data/local/home/.py_backup/config.ini`:

```
        SM64,True,/media/data/local/home/.sm64-port
```

It is important to respect the indentation. Then open Py Backup and you will see a new entry for SM64.
